.github/workflows/Tests.yml

```diff
@@ -245,12 +245,12 @@
       if: ${{ runner.os }} == "Linux"
       shell: bash
       run: |
-        if [ "$OS" == "ubuntu-latest" ] ; then ./cc-test-reporter after-build --exit-code 0 || true ; else echo "SKIP code climate" ; fi ;
+        ./cc-test-reporter after-build --exit-code 0 || true ;
```

Comment:
🙈 this will indeed need to be corrected so it only runs on linux or darwin, but the implementation is over in the template repository [reactive-firewall/python-repo](https://github.com/reactive-firewall/python-repo/)

:thinking: Please summarize this discussion so-far, for posterity, and generate a possible plan for improving and re-instating the checks by OS. Then open a new ticket, with the title "re-instating conditional use of codeclimate and deepsource tools in CI", use the generated summary and possible plan for the new issue's description.